source/_includes/installation/core.md

```diff
@@ -33,7 +33,7 @@ sudo apt-get upgrade -y
 Install the dependencies:
 
 ```bash
-sudo apt-get install -y python3 python3-dev python3-venv python3-pip libffi-dev libssl-dev libjpeg-dev zlib1g-dev autoconf build-essential libopenjp2-7 libtiff5 libturbojpeg0-dev tzdata
+sudo apt-get install -y python3 python3-dev python3-venv python3-pip libffi-dev libssl-dev libjpeg-dev zlib1g-dev autoconf build-essential libopenjp2-7 libtiff5 libturbojpeg0-dev tzdata cargo ndg-httpsclient pyopenssl pyasn1
```

Comment:
Why are these added? This is system-dependent?

This is not needed on the Debian (which is the reference).

Reply:
I am on Armbian latest which is based on Debian bullseye, it complains lack of Rust Complier even install it manually but works fine if cargo installed.

ndg-httpsclient pyopenssl and pyasn1 are used to resolve a SSL problem I encountered that can not download from GitHub.com.